src/cloud/trouble/error-codes.md

```diff
@@ -0,0 +1,115 @@
+---
+group: cloud-guide
+title: Error message reference for ece-tools
+---
+
+{% assign errors = site.data.cloud-error-messages %}
+
+This {{site.data.var.ct}} error message reference provides information to resolve errors that occur during the {{site.data.var.ece}} deployment process.
+
+Error messages are categorized by deployment stage–build, deploy, and post-deploy. Each section provides a list of associated errors with the following information for each error:
```

Comment:
```suggestion
Error messages are categorized by deployment stage: build, deploy, and post-deploy. Each section provides a list of associated errors with the following information for each error:
```

Also, you start this paragraph about the deployment steps, but then context switch to "Sections", which I assumes means the tables below. I would clarify or change section to stage.